surfactant/infoextractors/elf_file.py

```diff
@@ -79,82 +80,84 @@ def extract_elf_info(filename: str) -> object:
+                            file_details["elfDependencies"].append(tag.needed)
+                        elif tag.entry.d_tag == "DT_RPATH":
+                            # Library rpath
+                            file_details["elfRpath"].append(tag.rpath)
+                        elif tag.entry.d_tag == "DT_RUNPATH":
+                            # Library runpath
+                            file_details["elfRunpath"].append(tag.runpath)
+                        elif tag.entry.d_tag == "DT_SONAME":
+                            # Library soname (for linking)
+                            file_details["elfSoname"].append(tag.soname)
+                        elif tag.entry.d_tag == "DT_FLAGS":
+                            # Dynamic Flags, DT_FLAGS
+                            dt_flags_entry: Dict[str, Any] = {}
+                            dt_flags_entry["value"] = hex(tag.entry.d_val)
+                            # $ORIGIN processing is required
+                            dt_flags_entry["DF_ORIGIN"] = bool(
+                                tag.entry.d_val & ENUM_DT_FLAGS["DF_ORIGIN"]
+                            )
+                            # Perform complete relocation processing (part of Full RELRO)
+                            dt_flags_entry["DF_BIND_NOW"] = bool(
+                                tag.entry.d_val & ENUM_DT_FLAGS["DF_BIND_NOW"]
+                            )
+                            file_details["elfDynamicFlags"].append(dt_flags_entry)
+                        elif tag.entry.d_tag == "DT_FLAGS_1":
+                            # Dynamic Flags, DT_FLAGS_1 (custom entry first added by binutils)
+                            dt_flags_1_entry: Dict[str, Any] = {}
+                            dt_flags_1_entry["value"] = hex(tag.entry.d_val)
+                            # Position-Independent Executable file
+                            dt_flags_1_entry["DF_1_PIE"] = bool(
+                                tag.entry.d_val & ENUM_DT_FLAGS_1["DF_1_PIE"]
+                            )
+                            # Perform complete relocation processing
+                            dt_flags_1_entry["DF_1_NOW"] = bool(
+                                tag.entry.d_val & ENUM_DT_FLAGS_1["DF_1_NOW"]
+                            )
+                            # $ORIGIN processing is required
+                            dt_flags_1_entry["DF_1_ORIGIN"] = bool(
+                                tag.entry.d_val & ENUM_DT_FLAGS_1["DF_1_ORIGIN"]
+                            )
+                            # Ignore the default library search path
+                            dt_flags_1_entry["DF_1_NODEFLIB"] = bool(
+                                tag.entry.d_val & ENUM_DT_FLAGS_1["DF_1_NODEFLIB"]
+                            )
+                            file_details["elfDynamicFlags1"].append(dt_flags_1_entry)
+        except ELFParseError as e:
+            logger.warning(f"Error while parsing ELF sections: {repr(e)}")
```

Comment:
```suggestion
            logger.warning(f"Error while parsing ELF sections in {filename}: {repr(e)}")
```

We should also make sure it is clear which file caused the parsing errors.